Hotel/Room.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hotel
+{
+    internal class Room
+    {
+        public int IdRoom { get; set; }
+
+        public int NumberRoom { get; set; }
```

Comment:
Is it an actual situation when there are rooms with the same number in a hotel? If nope, you shouldn't use id room, because room id is a number.